build/templates/_mock_helper.py.mako

```diff
@@ -30,7 +30,7 @@ class MockFunctionCallError(Exception):
 class SideEffectsHelper(object):
     def __init__(self):
         self._defaults = {}
-% for func_name in sorted(helper.filter_codegen_functions(functions)):
+% for func_name in sorted(functions):
```

Comment:
We didn't need any filtering here or later because we already filter functions at line 15.